clang/bindings/python/clang/cindex.py

```diff
@@ -1336,7 +1336,7 @@ def __repr__(self):
 CursorKind.OMP_TEAMS_DISTRIBUTE_DIRECTIVE = CursorKind(271)
 
 # OpenMP teams distribute simd directive.
-CursorKind.OMP_TEAMS_DISTRIBUTE_DIRECTIVE = CursorKind(272)
+CursorKind.OMP_TEAMS_DISTRIBUTE_SIMD_DIRECTIVE = CursorKind(272)
```

Comment:
This was a duplicate with variant 271. new name taken from https://clang.llvm.org/doxygen/Index_8h_source.html, line 2011

Reply:
LGTM, but worth an entry in release notes under "potentially breaking changes".